### PR TITLE
Split visualizations into chunks

### PR DESCRIPTION
**Pull request by plesiecki**
https://github.com/allegro/turnilo/issues/845

### CODE REVIEW

File: src/client/visualizations/index.ts [hunk 1]
```diff
@@ -16,24 +16,19 @@
...
+  "line-chart": () => import(/* webpackChunkName: "line-chart" */ "./line-chart/line-chart"),
+  "bar-chart": () => import(/* webpackChunkName: "bar-chart" */ "./bar-chart/bar-chart"),
+  "heatmap": () => import(/* webpackChunkName: "heatmap" */ "./heat-map/heat-map"),
+  "grid": () => import(/* webpackChunkName: "grid" */ "./grid/grid"),
+  "scatterplot": () => import(/* webpackChunkName: "scatterplot" */ "./scatterplot/scatterplot")
 };
 
-export function getVisualizationComponent({ name }: VisualizationManifest) {
+export function getVisualizationComponent({ name }: VisualizationManifest): () => Promise<{
+  default: React.ComponentType<any>;
```
Comment (adrianmroz-allegro) [inline src/client/visualizations/index.ts:31]: `any` ⚠️

Comment (plesiecki) [reply src/client/visualizations/index.ts]: gone

**Review by mkuthan — COMMENTED** [review]
I expected more changes from splitting visualisations

**Review by adrianmroz-allegro — APPROVED** [review]

File: config/webpack.prod.js [hunk 2]
```diff
@@ -53,7 +53,16 @@ const prodConfig = {
       'process.env.NODE_ENV': JSON.stringify('production')
     }),
   ],
-  stats: 'minimal'
+  optimization: {
```
[the remark this answers is not in the document]
Comment (plesiecki) [reply config/webpack.prod.js:56]: Thanks for pointing that out.

**Pull request merged**